solutions/secure/catalogValidationValues.json.template

```diff
@@ -0,0 +1,6 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "region": "us-south"
+  "resource_group_name": $RG_NAME,
```

Comment:
Don't use `$RG_NAME` here - its needs to be a unique name since by default it will create the group. Just use `$PREFIX`